README.md

```diff
@@ -1 +1,210 @@
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [-hello-microvers] <a name="about-project"></a>
+
+> This project was part of Microverse week one tasks.
+It gives the student an opportunity to learn and do with HTML, CSS, and Github.
+
+**[Hello Microvers]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+- HTML
+- CSS
+- Linters
+### Tech Stack <a name="tech-stack"></a>
+
+> The tech stack used in this project include HTML,CSS and Github.
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> This project contain the features listed below.
+
+- **[HTML]**
+- **[CSS]**
+- **[Github]**
+- **[Linters]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+doesn't require installation.
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+Open index.html with a browser
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+For now no test is implemented.
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+Copying the project folder's contents to the root of 
+your server.
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/tes-nat)
+- Twitter: [@twitterhandle](https://twitter.com/tes_nat23)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/natnael-tesfaye-96488a264/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> The features I will add to the project.
+
+- [ ] **[javascript]**
+Another Features I will add to the project are currently undecided
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
```

Comment:
[OPTIONAL]
- [ ] Kindly remove these comments since it was just there for your guidance, keeping them will make your `README` look unprofessional.